src/ui/gameday/plays.rs

```diff
@@ -129,9 +134,17 @@ fn format_runs(play: &PlayResult, selected_at_bat: Option<u8>) -> Span {
 }
 
 /// If runs were scored display the new score.
-fn format_score(play: &PlayResult) -> Span {
+fn format_score<'a>(
+    play: &'a PlayResult,
+    team_abbreviations: (&'static str, &'static str),
+) -> Span<'a> {
     if play.is_scoring_play {
-        Span::raw(format!(" {}-{}", play.away_score, play.home_score))
+        let (home, away) = team_abbreviations;
+        Span::raw(format!(
+            " ({} {}, {} {})",
```

Comment:
small nit - what if these were `[` `]` instead of parentheses? I kinda like that look a little better

Reply:
updated!